spa/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "spa",
```

Comment:
Maybe not matter, but anyway update the name as well;

Reply:
Couldn't think of a better name, so keeping it as it is  for now